test/IISIntegration.FunctionalTests/Inprocess/ResponseInvalidOrderingTests.cs

```diff
@@ -2,12 +2,14 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Server.IntegrationTesting;
 using Microsoft.AspNetCore.Testing.xunit;
 using Xunit;
 
 namespace Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests
 {
     [Collection(IISTestSiteCollection.Name)]
+    [SkipIfISExpressSchemaMissingInProcess]
```

Comment:
Why do we need to copy it everywhere now?

Reply:
It was incorrect before, it was applied globally and would skip even Out-of-proc tests.

Reply:
Anyway to add it to make it specific to the fixture rather than the test?

Reply:
Nope.

Reply:
Pavel's proposal was to split the test projects. You two can look at that later.